common/lib/container-definitions/src/deltas.ts

```diff
@@ -138,6 +138,8 @@ export interface IDeltaManager<T, U> extends IEventProvider<IDeltaManagerEvents>
     /** Flag to indicate whether the client can write or not. */
     readonly active: boolean;
 
+    readonly connectionMode?: ConnectionMode;
```

Comment:
We do not expose currently that info to runtime. So I wonder why and where it is needed?
Based on PR, it's used in test, but test code casts to any, so it does not use this new definition.

If it's needed (let's say in tests), I'd suggest go route of separate interface (like IDeltaSender) - something that is implented by DeltaManager class, but not part of IDeltaManager

Reply:
I was talking with @anthony-murphy about this, and I think the conclusion was that `connectionMode` should be something on the `IDeltaManager`, adding it as a separate interface could work as well. If we were to do it, then where would it be proper to add this, in `container-definitions`?